test/integration/targets/module_utils/library/test_alias_deprecation.py

```diff
@@ -4,7 +4,8 @@
 __metaclass__ = type
 
 from ansible.module_utils.basic import AnsibleModule
-from ansible.module_utils.facts import data
+# overridden
+from ansible.module_utils.ansible_release import data
```

Comment:
Why is this being changed?

Reply:
Due to it previously overriding `ansible.module_utils.facts` the `setup_remote_tmp_dir` role will not run, as it fails to import facts for determining the OS type.  So I renamed the file to something less likely to be used by any of our modules.

Reply:
Could you add that to the comment for future reference?

Reply:
Updated, not in this file, but to the overridden file, and I updated some other references to `facts.py`